### PR TITLE
Fix/liwo 215

### PR DESCRIPTION
**Pull request by SiggyF**
Fix https://issuetracker.deltares.nl/browse/LIWO-215

### CODE REVIEW

File: src/lib/export-map-zip.js [hunk 1]
```diff
@@ -1,10 +1,11 @@
 import downloadBlob from './download-blob'
 import mapConfig from '../map.config'
 
-const apiBase = mapConfig.services.WEBSERVICE_URL
 const headers = { 'Accept': '*/*', 'Content-Type': 'application/json' }
 
-export default function ({ layers, name }) {
+export default async function ({ layers, name }) {
```
Comment (CindyvdVries) [inline src/lib/export-map-zip.js]: No comments in this file?

File: src/lib/leaflet-directive.js [hunk 2]
```diff
@@ -1,4 +1,4 @@
-import layerFactory from './leaflet-utils/layer-factory'
+import createLayer from './leaflet-utils/layer-factory'
```
Comment (CindyvdVries) [inline src/lib/leaflet-directive.js]: Should file also be renamed?

File: src/lib/load-layersets.js [hunk 3]
```diff
@@ -86,14 +88,16 @@ export async function loadLayerSetById (id, options) {
...
 }
 
-export function loadLayerSets () {
+export async function loadLayerSets () {
   // we don't need to login anymore, but the function is still used to
   // get the list of public maps
   const body = JSON.stringify({
-    username: 'marko@voorhoede.nl',
-    password: 'DikkeDoei123',
+    username: 'anonymous@rws.nl',
```
Comment (CindyvdVries) [inline src/lib/load-layersets.js]: un and pw specified in code?

Comment (SiggyF) [reply src/lib/load-layersets.js]: Not anymore. I removed them. The service is anonymous.

File: src/lib/geoserver-url.js [hunk 4]
```diff
@@ -13,7 +13,7 @@ export default function ({
   if (!service || !request) {
     return undefined
   }
-
+  let services = mapConfig.getServices()
```
Comment (SiggyF) [inline src/lib/geoserver-url.js]: should have been `await`, fixed in other branch.

File: src/lib/leaflet-directive.js (hunk 2 above)
Comment (SiggyF) [reply src/lib/leaflet-directive.js]: The file exposes several functions,  the default is called `createLayer`.  It also creates clusters so for now it's ok...

**Pull request merged**